manifests/init.pp

```diff
@@ -101,9 +101,16 @@
     ensure => $package_ensure,
   }
 
+  $config_path = '/etc/gitlab-runner/config.toml'
```

Comment:
Let's make this a module parameter so users can change this value when calling `class {'gitlab_ci_runner'`

Reply:
I added the variable as parameter. There is also a test for this in place.